ProcessLib/HydroMechanics/HydroMechanicsFEM.h

```diff
@@ -76,7 +77,8 @@ struct IntegrationPointData final
     {
         auto&& solution = solid_material.integrateStress(
             t, x_position, dt, eps_prev, eps, sigma_eff_prev,
-            *material_state_variables);
+            *material_state_variables,
+            MaterialLib::PhysicalConstant::RoomTemperature, 0.);
```

Comment:
I'd move it out into configuration file, because there are many different standards for temperature.
Also when using dimensionless form of equations, the absolute value is a no-go.

Reply:
I also understand, that the last 0 function argument is pressure, but 0Pa pressure is not used in any standard.
In *M we explicitly do have the pressure of the system.

Reply:
Do you mean to set it as an input parameter?

Reply:
yes, like the "reference_temperature" in TM.

Reply:
Now,  reference_temperature is set an optional input, and reference_fluid_pressure is dropped because it is only for some special models for unsaturated flow processes.